modules/restore-add-master.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-add-master_{context}"]
+= Adding a master host back to the etcd cluster
+
+Follow these steps to add a master host back to the etcd cluster. This procedure assumes that you previously removed the master host from the cluster and that its TLS certificates are valid.
+
+// TODO: Ask why it can't be a completely new member?
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* SSH access to the master host to add to the etcd cluster.
+* The IP address of an existing active etcd member.
+
+.Procedure
+
+. Access the master host to add to the etcd cluster.
+
+. Run the `etcd-member-add.sh` script.
++
+Pass in two parameters to the `etcd-member-add.sh` script: IP address of an existing etcd member and the name of the etcd member to add.
++
+----
+[core@ip-10-0-128-73 ~]$ sudo -E etcd-member-add.sh 10.0.128.73 \ etcd-member-ip-10-0-147-172.us-east-2.compute.internal
```

Comment:
@retroflexer Can you verify that this prompt of `[core@ip-10-0-128-73 ~]` is okay here (and two other instances in the PR)?

Your draft had used `[core@ip-10-0-143-125 ~]`, but that one never shows up in the example member lists used, so I chose one that did.

Reply:
No. Actually this is very important. We need to emphasize that the add command should be run on the same master node of the member being added. The manifests and the environment needed for the new member is generated on its master node. 

 So, it should have been:
[core@ip-`10-0-147-172` ~]

Reply:
Having the prompt as `[core@ip-10-0-128-73 ~]` is definitely misleading and wrong.

Reply:
^^ This should be a big note. If they run this script on the wrong master they could lose quorum.

Reply:
Okay, I see. So for adding back, they need to be on the host being added, so the prompt here should be `[core@ip-10-0-147-172 ~]`.

But what about for when removing, you don't to be on the host you're removing, do you? That's why I had chosen `[core@ip-10-0-128-73 ~]` there. Let me know about this one.

Reply:
Yup. remove can be performed from any of the hosts. 

But the BIG NOTE should alert about the `add`. Add must be performed from its own master node, as the files that are generated as part of the procedure are needed to be on the master node for the member to start successfully.